lib/whitehall_importer.rb

```diff
@@ -23,9 +25,26 @@ def self.create_migration(organisation_content_id, document_type)
     end
   end
 
-  def self.import_and_sync(whitehall_document)
-    whitehall_import = WhitehallMigration::DocumentImport.create!(
-      whitehall_document_id: whitehall_document["id"],
+  def self.queue_migration_jobs(whitehall_migration)
+    documents = WhitehallMigration::DocumentImport.where(whitehall_migration_id: whitehall_migration.id)
+    documents.each do |document|
+      WhitehallDocumentImportJob.perform_later(document)
+    end
+  end
+
+  def self.mark_migration_completed(whitehall_migration_document_import)
+    running_states = %w(pending importing imported syncing)
+    whitehall_migration = WhitehallMigration.find(whitehall_migration_document_import.whitehall_migration_id)
+    if WhitehallMigration::DocumentImport.where(whitehall_migration_id: whitehall_migration.id, state: running_states).empty?
+      whitehall_migration.update!(
+        end_time: Time.current,
+      )
```

Comment:
Minor, this could be a single line.

Reply:
Moved this onto a single as part of the change to scope `in_progress`